luigi/contrib/azurebatch.py

```diff
@@ -415,42 +441,58 @@ class AzureBatchTask(luigi.Task):
     :param pool_vm_size (str): size of vm to use for the batch job
 
     """
+
     batch_account_name = luigi.Parameter()
     batch_account_key = luigi.Parameter()
+    batch_account_url = luigi.Parameter()
     storage_account_name = luigi.Parameter()
     storage_account_key = luigi.Parameter()
-    input_path = luigi.Parameter()
-    output_path = luigi.Parameter()
+    input_path = luigi.Parameter(default=" ")
+    command = luigi.Parameter(default="echo Hello World")
+    output_path = luigi.Parameter(default=" ")
     pool_id = luigi.Parameter(default=TASK_POOL_ID)
     job_id = luigi.Parameter(default=TASK_JOB_ID)
     pool_node_count = luigi.IntParameter(default=TASK_POOL_NODE_COUNT)
     pool_vm_size = luigi.Parameter(default=TASK_POOL_VM_SIZE)
     kwargs = luigi.DictParameter(default={})
+    container_name = "luigitargetdata"
 
     def run(self):
         bc = AzureBatchClient(
-                self.batch_account_name,
-                self.batch_account_key,
-                self.batch_account_url,
-                self.storage_account_name,
-                self.storage_account_key,
-                self.output_path,
-                self.pool_id,
-                self.job_id,
-                self.pool_node_count,
-                self.pool_vm_size,
-                self.kwargs)
-
-        input_files = [bc.upload_file_to_container(self.container_name, file_path) for file_path in os.listdir(self.input_path)]
+            self.batch_account_name,
+            self.batch_account_key,
+            self.batch_account_url,
+            self.storage_account_name,
+            self.storage_account_key,
+            self.output_path,
+            self.pool_id,
+            self.job_id,
+            self.pool_node_count,
+            self.pool_vm_size,
+            # self.kwargs,

```

Comment:
Clean this up.

Reply:
This needs to be fixed. I have commented it as we do not need it now

Reply:
ok